src/config.rs

```diff
@@ -266,3 +272,57 @@ impl SizeTieredCompactionSchedulerOptions {
         }
     }
 }
+
+/// Garbage collector options.
+#[derive(Clone)]
+pub struct GarbageCollectorOptions {
+    /// Garbage collection options for the manifest directory.
+    pub manifest_options: Option<GarbageCollecterDirectoryOptions>,
```

Comment:
how about remove the Option here like `pub manifest_options: GarbageCollecterDirectoryOptions`?

these `GarbageCollecterDirectoryOptions` fields seems all had reasonable defaults

Reply:
I wanted to allow developers to disable GC for the manifest, wal, and compacted. If I eliminate the `Option` here, I'd have to move the options into the `min_age` and `poll_interval` variables. That felt clunky, since the on/off switch should be atomic for each. If we end up with min_age set but not poll_interval, that's a weird state to be in.